app/ide-desktop/lib/dashboard/src/services/LocalBackend.ts

```diff
@@ -21,6 +90,66 @@ function ipWithSocketToAddress(ipWithSocket: projectManager.IpWithSocket) {
   return backend.Address(`ws://${ipWithSocket.host}:${ipWithSocket.port}`)
 }
 
+// ======================================
+// === Functions for manipulating ids ===
+// ======================================
+
+/** Create a {@link backend.DirectoryId} from a path. */
```

Comment:
Just to help me understand -- what is a `path` in this case? Is it an actual path on the filesystem? If it is, perhaps `Path` should be typed?